### PR TITLE
[DocumentIntelligence] Update SDK samples to print table cell values and print paragraphs by reading order

### PR DESCRIPTION
**Pull request by TakaValley**
# Description
1.add SDK samples for extracting table cell values
2.add samples for sorting paragraphs by right reading order

# All SDK Contribution checklist:
- [X] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by TakaValley** [comment]
> @TakaValley please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="MSFT"]

**Comment by TakaValley** [comment]
> @TakaValley please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree MSFT

**Comment by TakaValley** [comment]
> @TakaValley the command you issued was incorrect. Please try again.
> 
> Examples are:
> 
> ```
> @microsoft-github-policy-service agree
> ```
> 
> and
> 
> ```
> @microsoft-github-policy-service agree company="your company"
> ```

@microsoft-github-policy-service agree company="Microsoft"

**Comment by toothache** [comment]
@YalinLi0312 , can you help with the review?

File: sdk/documentintelligence/azure-ai-documentintelligence/samples/helper/utils.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
```
Comment (toothache) [inline sdk/documentintelligence/azure-ai-documentintelligence/samples/helper/utils.py:1]: @YalinLi0312 

Our PM got customer feedback that we should have more readable print result. That's why we introduced this utility file and share it among the samples. Let us know your thoughts on this. Thanks!

**Review by YalinLi0312 — COMMENTED** [review]
@TakaValley  Can you remove the helper as we want each sample to be self-contained? After that, it looks good to me.

**Comment by TakaValley** [comment]
> @TakaValley Can you remove the helper as we want each sample to be self-contained? After that, it looks good to me.
Thanks. Fixed as suggested

File: sdk/documentintelligence/azure-ai-documentintelligence/samples/helper/utils.py (hunk 1 above)
Comment (TakaValley) [reply sdk/documentintelligence/azure-ai-documentintelligence/samples/helper/utils.py]: Fixed it. Thanks.

**Pull request merged**